### ISSUE TITLE
[4.x]: SQL error during v4 upgrade migration

### ISSUE THREAD

**Issue by jmauzyk** [craftcms/commerce#3128]
### Description

Getting the SQL error below when running the `craft commerce/upgrade` command.

### Steps to reproduce

1. Run `craft commerce/upgrade`.
2. Respond to initial prompts regarding address fields.

### Expected behavior

Expected to run migration successfully

### Actual behavior

Fails with SQL error
```
Creating a user for every customer…
Error: SQLSTATE[HY000]: General error: 7 number of parameters must be between 0 and 65535
The SQL being executed was: DELETE FROM "commerce_customers" WHERE "id" IN (42108531, 6, 13, ... 37137672, 38815727, 37434134)
```

### Craft CMS version

4.4.5

### Craft Commerce version

4.2.6

### PHP version

8.2.4

### Operating system and version

Linux 4.15.0-65-generic

### Database type and version

PostgreSQL 14.7

### Image driver and version

Imagick 3.7.0 (ImageMagick 6.9.7-4)

**Comment by pdaleramirez** [comment]
@jmauzyk It seems to be that the database commerce_customers.id column hit the length limit. You could test modifying the column changing from `INT` to `BIGINT` type, then rerun the upgrade command.

**Comment by pdaleramirez** [comment]
@jmauzyk, can you post the stack trace for this issue? We’ll try to find out which migration file is causing the problem.

**Comment by lukeholder** [comment]
I have a potential fix for this in #3133 - we just need to do some testing before rolling it into a release.

Thanks.

**Comment by pdaleramirez** [comment]
@jmauzyk Would it be possible to send us your database to **support@craftcms.com**? We could try out your database with the fix.
